spec/feature/cards_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature "Basic training" do
+  scenario "Try to learning new cards" do
+    visit "/"
+
+    click_link "тренировку"
+    expect(page).to have_content 'Проверка'
+    find("#review_user_translation").set "TestTest"
+    find("input[value='Проверить']").click
+    expect(page).to have_content 'переведено с ошибкой'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.